src/Components/ArtsyShippingEstimate.tsx

```diff
@@ -405,32 +403,32 @@ const UNFRAMED_CATEGORY_MAP = {
 } as const
 
 interface ShippableArtwork {
-  depthCm?: number
-  diameterCm?: number
-  framedDepth?: number
-  framedDiameter?: number
-  framedHeight?: number
-  framedMetric?: string
-  framedWidth?: number
-  heightCm?: number
-  isFramed: boolean
-  listPrice: {
-    major?: number
+  depthCm?: number | null
```

Comment:
Made the mirrored type here a better mirror. In retrospect we could probably have used a relay type for this prop if we don't need to support edition sets too. That might be some cleanup worth taking on later.